web/templates/poll_modal_option.hbs

```diff
@@ -1,7 +1,7 @@
 <li class="option-row">
     <i class="zulip-icon zulip-icon-grip-vertical drag-icon"></i>
     <input type="text" class="poll-option-input modal_text_input" placeholder="{{t 'New option' }}" />
-    <button type="button" class="button rounded small btn-secondary delete-option" title="{{t 'Delete' }}">
+    <button type="button" class="button rounded small button-secondary delete-option" title="{{t 'Delete' }}">
```

Comment:
`btn-secondary`/`button-secondary` doesn't seem to have any styles; can we just delete it as unused?

Reply:
This is covered in #32415.

Reply:
Fixed in #32421.